tests/containers/suma_containers.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test SUMA container images:
+#   The scope of the testing is minimal and covers pulling and running 
+#   container images from the internal registry.suse.de
+#   
+# The container images tested are:
+#   - proxy-httpd
+#   - proxy-salt-broker
+#   - proxy-squid
+#   - proxy-ssh
+#   - proxy-tftpd
+#
+# The testing of proxy-charts is limited to pulling the tar from the registry to
+# and inspect the content to ensure Charts is available. 
+# Install or deployment is not covered in this scenario.
+#   
+# In depth testing of all the images  is already done by the SUMA QE team
+#
+# Maintainer: Maurizio Galli <maurizio.galli@suse.com>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+
+sub run {
+    select_serial_terminal;
+
+    my $image = get_required_var('CONTAINER_IMAGE_TO_TEST');
+
+    if (check_var('CONTAINER_SUMA', 'image')) {
+        script_retry("podman pull $image", timeout => 300, delay => 60, retry => 3);
+        assert_script_run("podman run --rm $image cat /etc/os-release");
+    }
+    elsif (check_var('CONTAINER_SUMA', 'chart')) {
+        script_retry("helm pull $image", timeout => 300, delay => 60, retry => 3);
+        assert_script_run("tar -tf proxy-*.tgz && rm proxy-*.tgz");
+    }
+}
+
+sub test_flags {
+    return {fatal => 1};
```

Comment:
Do we really want to mark this test module as fatal, i.e. the whole test run will be stopped and other tests won't be executed?

Reply:
The tests check that the container image exists and runs. I think it's ok to stop here if  either of these two basic tests fail. Or did I not understand your remark?

Reply:
`fatal => 1` stops the whole test run. So if this is expected to be a short test run that consists only of this test module, this is IMHO not needed. If this would be in a larger test run, where e.g. you would execute this next to other container tests, then all of the following ones would be skipped.

So from my POV - in the former case it would not matter, in the latter case it would be undesired. So I'd argue, unless necessary, this should not be there.

WDYT?

Reply:
I included it because it was in the Rancher test I based this from. If it's not needed I'm ok to remove it, I don't know know openQA well enough to have a strong opinion on this :-)

Reply:
I removed the fatal => 1 part